src/parser/parser_state.cpp

```diff
@@ -157,7 +157,7 @@ Term ParserState::bindVar(const std::string& name,
                           bool doOverload)
 {
   Trace("parser") << "bindVar(" << name << ", " << type << ")" << std::endl;
-  Term expr = d_solver->mkConst(type, name);
+  Term expr = d_solver->getTermManager().mkConst(type, name);
```

Comment:
This is relatively performance intensive code, can you cache a term manager pointer in `ParserState` in this class and use it here?